lib/rules/strict.js

```diff
@@ -17,7 +17,9 @@ var messages = {
     global: "Use the global form of \"use strict\".",
     multiple: "Multiple \"use strict\" directives.",
     never: "Strict mode is not permitted.",
-    unnecessary: "Unnecessary \"use strict\" directive."
+    unnecessary: "Unnecessary \"use strict\" directive.",
+    unnecessaryInModules: "Unnecessary \"use strict\" directive, because inside of modules.",
```

Comment:
After seeing the documentation, yes, it is more clear what is meant. But I would still consider those the same "form", just in different position. Something like "this directive should be moved to function position" and "this directive should be move to the top level" would be far better. Additionally, the option name "global" doesn't really make sense. The only way "global" is related to "top level" is that variables declared there when the program is parsed as a Script are added to the global object. Anyway, I'd rather not derail this PR, so feel free to open an issue if you agree.